cannot use an integer argument

Using this query

```
- uses: octokit/graphql-action@v2.x
        id: doc_info
        with:
          query: |
            query doc_info($owner: String!, $repo: String!, $pr_number: Int!) {
              repository(owner: $owner, name: $repo) {
                pullRequest(number: $pr_number) {
                 commits(first: 10) {
                   edges {
                     node {
                       commit {
                         oid
                         message
                       }
                     }
                   }
                 }
               }
              }
            }
          owner: ${{ github.event.repository.owner.name }}
          repo: ${{ github.event.repository.name }}
          pr_number: $ {{ github.event.pull_request.number }}
```

I am getting this error:

```
##[error]Variable $pr_number of type Int! was provided invalid value
##[error]Node run failed with exit code 1
```

How can I deal with it?
I don't see any casting to number method in the github [context expression documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions)

--- a/index.js
+++ b/index.js
@@ -34,6 +34,7 @@ function getAllInputs() {
 
     const inputName = key.substr("INPUT_".length).toLowerCase();
     result[inputName] = value;
+    result[inputName] = result[inputName] == parseInt(result[inputName], 10) ? parseInt(result[inputName], 10) : result[inputName];
     return result;
   }, {});
 }

